source/includes/steps-create-agg-pipeline.yaml

```diff
@@ -50,5 +50,22 @@ content: |
 
          .. figure:: /images/compass/agg-builder-exclude-stage-example.png
             :alt: Aggregation Builder exclude stage example
-...
+---
+title: Run Pipeline.
+level: 4
+ref: run-aggregation
+content: |
+  Click :guilabel:`Run` at the top right of the pipeline builder. 
```

Comment:
We may want to add an info/warning box to highlight the fact that when you hit `Run`, the aggregation runs against the entire collection.